tests/codec/codec_test.go

```diff
@@ -93,11 +91,6 @@ func newExample() *wasm.Module {
 
 func TestExampleUpToDate(t *testing.T) {
 	ctx := context.Background()
-	encoded := binary.EncodeModule(example)
-	// This means we changed something. Overwrite the example wasm file rather than force maintainers to use hex editor!
```

Comment:
this isn't necessary as a round-trip test happens below and serializing the same bytes is redundant